Reading List/BookController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Keri Levesque on 1/28/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+ 
+class BookController {
+    
+    var books: [Book] = [] {
+    didSet {
+            books.sort { $0.title < $1.title }
+        }
+    }
+    
+    var readBooks: [Book] {
+        let readBooks = books.filter { $0.hasBeenRead == true }
+        return readBooks
+    }
```

Comment:
Very good! Lines 20 and 21 can be rewritten as a single line of code like this:
`return books.filter({ $0.hasBeenRead })`